### DISCUSSION TITLE
I'll be back soon！

### DISCUSSION THREAD

**Discussion by Tnze** [Tnze/go-mc#172] [General]
@maxsupermanhd thank you very much for your use and feedback on this library. But I'm afraid I need some time before view your issues. When I'm interested, I'll check and fix these bugs step by step. I'm very sorry for your waiting🫠

**Comment by Tnze** [comment]
I'm focusing on my new projects of  ff14, a simulator & solver for its crafting system.

https://github.com/Tnze/ffxiv-best-craft

**Comment by maxsupermanhd** [comment]
soon when

**Comment by Tnze** [comment]
Well, the work will be signed in a few days
